tests/token_token_create_transaction.rs

```diff
@@ -0,0 +1,306 @@
+mod common;
+use common::setup_global;
+use hedera::{ Client, TokenCreateTransaction, TokenType, TokenSupplyType, TokenId, AccountId, PublicKey, TokenInfo, TokenInfoQuery, Key, KeyList, PrivateKey, Hbar, FractionalFee, FractionalFeeData, RoyaltyFee, RoyaltyFeeData, AnyCustomFee, FeeAssessmentMethod };
+use crate::common::TestEnvironment;
+use time::Duration;
+
+enum PublicKeyType {
+    PublicKey(PublicKey),
+    KeyList(KeyList),
+}
+
+struct UsedData {
+    name: String,
+    token_type: TokenType,
+    symbol: String,
+    decimals: u32,
+    initial_supply: u64,
+    account_id: AccountId,
+    public_key: PublicKeyType,
+    private_keys: Option<Vec<PrivateKey>>,
+    custom_fees: Vec<AnyCustomFee>,
+    max_supply: Option<u64>,
+    token_supply_type: TokenSupplyType,
+    default_freeze_status: bool,
+    auto_renew_period: Duration,
+    memo: String,
+}
+
+#[tokio::test]
+async fn test_create_fugible_token() {
+    let TestEnvironment { config, client } = setup_global();
+
+    let Some(operator) = &config.operator else {
+        log::debug!("skipping test due to lack of operator");
+        panic!("skipping test due to lack of operator");
```

Comment:
This panic is explicitly undesirable because we want to skip the test rather than fail it and the only way to really do that is to succeed.

This pattern is used by [Rust analyzer](https://github.com/rust-lang/rust-analyzer/blob/d42d55feaafa71e14521bbfe6e7011fbb41980f0/crates/ide/src/syntax_highlighting/tests.rs#L1073)